expression/builtin_math.go

```diff
@@ -33,6 +33,7 @@ import (
 var (
 	_ functionClass = &absFunctionClass{}
 	_ functionClass = &ceilFunctionClass{}
+	_ functionClass = &floorFunctionClass{}
```

Comment:
Please add definitions of floorFunctionClass and builtinFloorSig at the beginning of this file.